### PR TITLE
feat: add logging

### PR DESCRIPTION
**Pull request by gogolmariana**
Configured Logging in **settings.py**

Added logging to the views across different apps.

Added logging in permissions.py (in **users**/).
Screenshot 2024-10-11 at 18 35 02

### CODE REVIEW

File: forum/investors/views.py [hunk 1]
```diff
@@ -1,3 +1,12 @@
-from django.shortcuts import render
+from django.shortcuts import render, HttpResponse
```
Comment (k0drin) [inline forum/investors/views.py:1]: You import django.shortcuts render but unused it

File: forum/investors/views.py [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
-# Create your views here.
+logger = logging.getLogger(__name__)
+
+
+def investors(request):
+    try:
+        logger.info("Processing the request.")
+        return HttpResponse("Not implemented")
+    except Exception as e:
+        logger.error(f"Error occurred: {e}")
```
Comment (k0drin) [inline forum/investors/views.py:12]: No newline at end of file

File: forum/projects/views.py [hunk 3]
```diff
@@ -1,3 +1,12 @@
-from django.shortcuts import render
+from django.shortcuts import render, HttpResponse
```
Comment (k0drin) [inline forum/projects/views.py:1]: You import 'django.shortcuts.render' but unused it

File: forum/projects/views.py [hunk 4]
```diff
@@ -1,3 +1,12 @@
...
-# Create your views here.
+logger = logging.getLogger(__name__)
+
+
+def projects(request):
+    try:
+        logger.info("Processing the request.")
+        return HttpResponse("Not implemented")
+    except Exception as e:
+        logger.error(f"Error occurred: {e}")
```
Comment (k0drin) [inline forum/projects/views.py:12]: No newline at end of file

File: forum/startups/urls.py [hunk 5]
```diff
@@ -1,3 +1,6 @@
-urlpatterns = [
+from django.urls import path, include
```
Comment (k0drin) [inline forum/startups/urls.py:1]: You import 'django.urls.include' but unused it

File: forum/startups/urls.py [hunk 6]
```diff
@@ -1,3 +1,6 @@
...
+from . import views
 
+urlpatterns = [
+    path("", views.startups, name="startups"),
 ]
```
Comment (k0drin) [inline forum/startups/urls.py:6]: No newline at end of file

File: forum/startups/views.py [hunk 7]
```diff
@@ -1,3 +1,12 @@
...
-# Create your views here.
+logger = logging.getLogger(__name__)
+
+
+def startups(request):
+    try:
+        logger.warning("Processing the request.")
+        return HttpResponse("Not implemented")
+    except Exception as e:
+        logger.error(f"Error occurred: {e}")
```
Comment (k0drin) [inline forum/startups/views.py:12]: No newline at end of file

File: forum/users/views.py [hunk 8]
```diff
@@ -1,6 +1,15 @@
 from django.shortcuts import render, HttpResponse
```
Comment (k0drin) [inline forum/users/views.py:1]: You import 'django.shortcuts.render' but unused it

**Review by mehalyna — APPROVED** [review]

**Review by Chelakhovl — APPROVED** [review]

File: forum/investors/views.py (hunk 1 above)
Comment (gogolmariana) [reply forum/investors/views.py:1]: Left unused import as a placeholder for next development. When you create a new app, view contain _
from django.shortcuts import render_ by default.

**Review by k0drin — APPROVED** [review]

**Pull request merged**